### ISSUE TITLE
[Bug]: Broken .urlbutton / HyperlinkEngine in 2025 [Pyrevit5] [Revit2025]

### ISSUE THREAD

**Issue by tay0thman** [pyrevitlabs/pyRevit#2347]
### ✈ Pre-Flight checks

- [X] I **don't have SentinelOne** antivirus installed (see above for the solution)
- [X] I have **searched in the issues** (open and closed) but couldn't find a similar issue
- [X] I have **searched in the [pyRevit Forum](https://discourse.pyrevitlabs.io)** for similar issues
- [X] I already **followed the [installation troubleshooting guide](https://pyrevitlabs.notion.site/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7) thoroughly**
- [X] I am using the **latest pyRevit Version**

### 🐞 Describe the bug

Using Pyrevit 5.0.0.24227 WIP.
urlbutton doesn't work with Revit 2025, However, It works fine inside Revit 2023, 2024

### ⌨ Error/Debug Message

```shell
An error occurred trying to start process
' https://status.bluebeam.com/' with working directory
2025'. The system cannot
find the file specified.
at
System. Diagnostics. ProcessStartWithCreateP rocess(ProcessS..
startlnfo)
at System. Diagnostics. ProcessStart(ProcessStartlnfo
startlnfo)
at
PyRevitLabs.PyRevit.Runtime.HyperlinkEngine.Execute(Script..
runtime)
```

### ♻️ To Reproduce

1. Install WIP 5.0.0.24227
2. Start Revit 2025
3. click on a ulrbutton (like the Regex tool)

### ⏲️ Expected behavior

Clicking a urlbutton show open a url using the webbrowser.

### 🖥️ Hardware and Software Setup (please complete the following information)

```shell
Windows 11 + Windows Defender + Crowdstrike Falcon
Revit 2025
Pyrevit 5 WIP
Intel Processor + Nvidia GPU
```

### Additional context
@dosymep @sanzoghenzo 
_No response_

**Comment by jmcouffin** [comment]
https://github.com/pyrevitlabs/pyRevit/blob/f9fa0d296386d548e198324dd752fb7fb5cd7b58/pyrevitlib/pyrevit/runtime/HyperlinkEngine.cs

**Comment by jmcouffin** [comment]
would that help https://www.codeproject.com/Questions/5380961/How-do-I-fix-net-8-process-start-url-issue?

**Comment by jmcouffin** [comment]
or `Process.Start(new ProcessStartInfo("https://www.example.com") { UseShellExecute = true });`

**Comment by jmcouffin** [comment]
fixed by #2357

**Comment by jmcouffin** [comment]
tested in 2025 and 2021 with url, files, folderpath
all good
